xchange-stream-gemini/src/main/java/info/bitrich/xchangestream/gemini/dto/GeminiOrderbook.java

```diff
@@ -54,15 +54,35 @@ public void updateLevels(GeminiLimitOrder[] levels) {
     }
   }
 
-  public OrderBook toOrderbook() {
-    GeminiLimitOrder[] askLevels = asks.values().toArray(new GeminiLimitOrder[asks.size()]);
-    GeminiLimitOrder[] bidLevels = bids.values().toArray(new GeminiLimitOrder[bids.size()]);
+  public OrderBook toOrderbook(int maxLevels, Date timestamp) {
```

Comment:
this class cannot make up its mind is it an adapter or an DTO

can't we move this to GeminiAdapters?

Reply:
created GeminiAdaptersX